### DISCUSSION TITLE
NextJS build fails with almost no logs

### DISCUSSION THREAD

**Discussion by flotischoti** [vercel/community#4438] [Help] [answered]
### Summary

Hello,
I'm trying to build my NextJs 13.5.4 application on Vercel (Hobby Plan) but the build always fails. The only log I get is this:
```
[10:48:44.650] Running build in Washington, D.C., USA (East) – iad1 (Hive)
[10:48:44.745] Cloning github.com/secret/secret(Branch: master, Commit: b350bac)
[10:48:45.064] Cloning completed: 317.92ms
[10:48:45.103] Previous build cache not available
[10:48:45.254] Running "vercel build"
[10:48:45.693] Vercel CLI 32.4.1
[10:48:46.028] Installing dependencies...
[10:49:08.196] 
[10:49:08.196] added 442 packages in 22s
[10:49:08.197] 
[10:49:08.197] 128 packages are looking for funding
[10:49:08.197]   run `npm fund` for details
[10:49:08.225] Detected Next.js version: 13.5.4
```

Locally, `next build` runs successfully. I can't really use `vercel build` locally because of missing admin permission on my machine which are required to create symlinks which is why `vercel build` fails locally but much later then the last log line above.

Without meaningful logs, I'm more or less stuck. Any ideas what I can try?

### Steps to Reproduce

1) I connected my vercel project to my git repo
2) I push something on my master branch
3) A vercel build is triggered and fails

**Comment by amyegan** [comment]
Hey @flotischoti! I'm not getting that error with my test project. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)?

**Reply by flotischoti** [reply, marked as answer]
Hi @amyegan,
While trying to create a minimal reproducible example and gradually copying over files from my original project to a fresh NextJs project, I randomly got actual logs on Vercel after 20 or so builts that lead me into the right direction. My project had the following two issues:

1. For whatever reason I had copied the following snippet into my `next.config.js` which broke the built. Probably because I don't have a styles directory. I removed it.
```
sassOptions: {
    includePaths: [path.join(__dirname, 'styles')],
},
```

2. I forgot to generate Prisma client after installing dependencies. So I added the following script in `package.json`:
```"postinstall": "prisma generate"```

After these changes, the built runs successfully.

The logging situation during Vercel built however is less than satisfying.
